src/vmm/src/device_manager/legacy.rs

```diff
@@ -184,4 +174,16 @@ mod tests {
         .unwrap();
         assert!(ldm.register_devices(vm.fd()).is_ok());
     }
+
+    #[test]
+    fn test_error_messages() {
+        assert_eq!(
+            format!("{}", Error::BusError(devices::BusError::Overlap)),
+            "Failed to add legacy device to Bus: New device overlaps with an old device."
+        );
+        assert!(
+            format!("{}", Error::EventFd(std::io::Error::from_raw_os_error(0)))
+                .contains("Failed to create EventFd: "),
+        );
+    }
```

Comment:
These tests only existed to mark the `impl Display` block as covered for coverage reports. This is no longer needed (the `#[error]` lines are not considered "lines of code" by grcov), so we've agreed to remove these tests whenever switching from a manual implementation to `thiserror` [[1]].

[1]: https://github.com/firecracker-microvm/firecracker/issues/3278#issuecomment-1411801685

Reply:
> so we've agreed to remove these tests whenever switching from a manual implementation to thiserror

@roypat 
OK.
I reverted unit test implementaion.